pkg/kubectl/cmd/get/get.go

```diff
@@ -575,7 +576,9 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 	}
 	w.Flush()
 	if nonEmptyObjCount == 0 && !o.IgnoreNotFound && len(allErrs) == 0 {
-		fmt.Fprintln(o.ErrOut, "No resources found.")
+		buf := bytes.NewBuffer(nil)
+		fmt.Fprintln(buf, "No resources found.")
+		return fmt.Errorf("%v", buf.String())
```

Comment:
You could simplify this a bit: `return fmt.Errorf("No resources found.")`

Reply:
I'll refactor and push it together with the tests i'm fixing.

Reply:
returning an error here would mean an empty list would error, which is not correct